docs/wikibook.md

```diff
@@ -61,3 +62,244 @@ void playGame(ArrayList<Character> characters, ArrayList<Item> items, ArrayList<
+
+```diff
+public class MainWiki4 {
+   static final int INVENTORY = 1001;
+   static final int MINIMUM_ID_FOR_ITEM = 2001;
+   static final int ID_DIFFERENCE_FOR_OBJECT_IN_TWO_LOCATIONS = 10000;
+   
++   static final ArrayList<String> HELP_COMMAND_STRINGS = new ArrayList<String>(Arrays.asList(
++    	"go: Moves the player from one location to another. The go command needs to be followed by one of the six valid directions: north, south, east, west, up, down. (e.g. go north)",
++    	"get: Allows the player to pick up an object that is in the same location as they are. The get command needs to be followed by the name of the object that the player wants to get. If the object can be got then it is added to the player’s inventory and is no longer in its original location. (e.g. get torch)",
++   	"use: Allows the player to use an object that is in their location or their inventory. The use command needs to be followed by the name of the object that the player wants to use. (e.g. use silver key)",
++    	"examine: Provides the player with a more detailed description of an object or character in the same location as they are or an object in their inventory. The examine command needs to be followed by the name of the object or character that the player wants to examine. (e.g. examine red die)",
++    	"say: The player will speak, saying the text that follows the say command. (e.g. say hello)",
++    	"quit: Ends the game, quitting the program.",
++    	"read: Will display the text of objects that can be read. (e.g. read book)",
++    	"move: Allows the player to move an object that is in the same location as they are. The move command needs to be followed by the name of the object that the player wants to move. If the object can be moved then the result of moving the object will be displayed on the screen. (e.g. move bed)",
++    	"open: Allows the player to open an openable item that is in the same location as they are. (e.g. open siler door)",
++    	"close: Allows the player to close an closable item that is in the same location as they are. (e.g. close siler door)",
++    	"playdice: The player will play a dice game with another character. If the player wins then a list of the items that the losing character is carrying is displayed and the user chooses an item to take; if the player loses, the other character takes a random item from the player’s inventory. "
++    	"The playdice command needs to be followed by the name of a character to play the dice game with. The character and the player need to be in the same location and both need to have a die. (e.g. playdice guard)"));
+
+    		[']
+    		[']
+    		[']
+```
+
+```diff
+void playGame(ArrayList<Character> characters, ArrayList<Item> items, ArrayList<Place> places) {
+  [']
+  [']
+  [']
++        case "help":
++           Console.writeLine();
++           for (String cmd : HELP_COMMAND_STRINGS) {
++          	  Console.writeLine(cmd);
```

Comment:
Would it be easier to keep the command strings inline (no for loop), as they aren't used elsewhere?

Reply:
Probably, but I wrote this at god knows what time and it works, and doing it another way might confuse people lol.